lib/ripper-tags.rb

```diff
@@ -121,7 +121,12 @@ def self.process_args(argv, run = method(:run))
       if !file_list.empty? then options.files = file_list
       elsif !options.recursive then abort(optparse.banner)
       end
-      options.format ||= File.basename(options.tag_file_name) == "TAGS" ? "emacs" : "vim"
+      if options.tag_file_name.nil?
+        options.tag_file_name = options.format == 'emacs' ? 'TAGS' : 'tags'
```

Comment:
> Looks good. I think, however, that options.tag_file_name ||= syntax reads better and cleaner than the explicit options.tag_file_name.nil?. Same for options.format.

Fine, I will refactor tomorrow.

Why you think so longer line, and mixed with `||=`, `==` and `ternary` 
symbol is better to read?